Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: spiceai/README.md

The Spice.ai Cloud Platform has many datasets that can be used within Spice.  A valid login for the Spice.ai Cloud Platform is required to access the datasets.  Before beginning this quickstart, [link your GitHub account to Spice.ai](https://spice.ai/login) to get access to the platform.

**Step 1.** Initialize a Spice project:

```bash
spice init spiceai-demo
cd spiceai-demo
```

> **Step 2.** Log in to Spice.ai to store your access key and token in the spice runtime configuration.

Reviewer: ```suggestion
**Step 2.** Use `spice login` to store the Spice.ai Cloud Platform API Key and Token.
```

Using app your_user/your_app
```

> This will create a `.env` file in the `spiceai-demo` directory with the following content:

Reviewer: ```suggestion
A `.env` file is created in the `spiceai-demo` directory with the following content:
```
